staging/src/k8s.io/apiextensions-apiserver/pkg/registry/customresource/validator.go

```diff
@@ -37,8 +36,8 @@ import (
 type customResourceValidator struct {
 	namespaceScoped       bool
 	kind                  schema.GroupVersionKind
-	schemaValidator       *validate.SchemaValidator
-	statusSchemaValidator *validate.SchemaValidator
+	schemaValidator       apiservervalidation.SchemaCreateValidator
+	statusSchemaValidator apiservervalidation.SchemaCreateValidator
```

Comment:
nit, pre-existing: `apiservervalidation` is misleading. `apiextensionsvalidation` would be "right".

Reply:
renamed